src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,12 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
-        return new Response($this->serializer->serialize($exception, $format['key']), $exception->getStatusCode(), $headers);
+        $response = new Response(null, $exception->getStatusCode(), $headers);
+
+        if (!$this->debug && $response->isServerError()) {
+            $exception->setMessage(Response::$statusTexts[$response->getStatusCode()]);
```

Comment:
I'd say that we could create a new `HTTPException` if we shouldn't change the exception message. I didn't though about this but indeed it'd be better if (in production) you'd like to log the origin exception.